LICENSE.GPL.txt

```diff
@@ -0,0 +1,310 @@
+================================================
+ The PyInstaller community hooks licensing terms
+================================================
+
+The PyInstaller community hooks are licensed under the terms of the GNU
+General Public License as published by the Free Software Foundation;
+either version 2 of the License, or (at your option) any later version.
+
+Please note that this license only applies to files which have the following header:
```

Comment:
```suggestion
Please note that this license only applies to files which state it in the header, like this:
```
Anyway I would rephrase this to cover all files, even thos without header.

Reply:
Reworded.